docs/user-guide/work-packages/share-work-packages/README.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_navigation:
+  title: Share work packages
+  priority: 963
+description: How to share work packages in OpenProject.
+keywords: work package, share, share work packages
+---
+
+# Share work packages
+
+Since the 13.1 it is possible to share work packages with users that are not members of a project or are not yet registered on your instance. In the latter case a user will need to register in order to view the work package.
+
+> **Note**: Sharing work packages with non-member is an Enterprise add-on and can only be used with [Enterprise cloud](../../../enterprise-guide/enterprise-cloud-guide/) or  [Enterprise on-premises](../../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free community edition is easy and helps support OpenProject.
+
+To share a work package with a project non-member select the detailed view of a work package and click the **Share** button.
+
+![Share button in OpenProject work packages](openproject_user_guide_share_button_wp.png)
+
+A dialogue window will open, showing the list of all users, who this work package has already been shared with. If the work package has not yet been shared, the list will empty. 
+
+> **Note**: For sharing with non members to be available, a respective role needs to be assigned to the **Non member role**. 
+
+![List of users with access to a work package in OpenProject](openproject_user_guide_shared_with_list.png)
+
+If the list contains multiple users you can filter it by Type or Role. 
+
+Following user types are available as filters:
+
+![Filter list of users by user type](openproject_user_guide_sharing_member_type_filter.png)
+
+- Project member - returns all users that are project members
+- Not project member - returns all users that are not project members
+- Project group - returns all users that are members of the group, which includes the project
+- Not project group - returns all users that are not members of the group, which includes the project
+
+Following user roles are available as filters:
+
+![Filter list of users by user role](openproject_user_guide_sharing_member_role_filter.png)
+
+You can search for a user via a user name or an email address. You can either select an existing user from the dropdown menu or enter an email address for an entirely new user, who will receive an invitation to create an account on your instance.
```

Comment:
I would add "search for a user or group".

Reply:
Maybe is also worth saying that you can add more than user/group at the same time.